app/controllers/article_controller.rb

```diff
@@ -118,7 +118,9 @@ def eds_init
     config.show.sections.each do |_section, fields|
       fields.each do |field, options|
         config.add_show_field field.to_s, options
-        config.add_index_field field.to_s, options.except(:separator_options)
+        exceptions = :separator_options
+        if _section == 'Subjects' then exceptions = %i[separator_options helper_method] end
```

Comment:
This looks good to me. The only comment here is that `"_"` as a prefix to a variable name means that the variable is unused (i.e., rubocop will insist that you use underscores to mark unused variables). So we can rename `_section` to `section`.